### DISCUSSION TITLE
Conditional validation doesn't work at all

### DISCUSSION THREAD

**Discussion by RomkaLTU** [livewire/livewire#6359] [7. Bugs (archived)]
### Livewire version

2.12.6

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Write  conditional validation, for example:

```
'contract.epay_existing_customer' => ['required', 'boolean'],
'contract.epay_email' => ['required_if:contract.epay_existing_customer,1', 'email'],
'contract.epay_id' => ['required_if:contract.epay_existing_customer,0', 'string'],
```
This will not work, any conditional rule I tried that may be suitable here doesn't work. 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by harmoney-richardwyke** [comment]
Have you tried updating to prefix with `form.`
```
'contract.epay_email' => ['required_if:form.contract.epay_existing_customer,1', 'email'],
```
